zerver/lib/email_mirror.py

```diff
@@ -210,6 +212,27 @@ def send_zulip(sender: UserProfile, stream: Stream, topic: str, content: str) ->
     )
 
 
+def send_mm_reply_to_stream(
+    user_profile: UserProfile, stream: Stream, topic: str, body: str
+) -> None:
+    try:
+        check_send_message(
+            sender=user_profile,
+            client=get_client("Internal"),
+            message_type_name="stream",
+            message_to=[stream.id],
+            topic_name=topic,
+            message_content=body,
+        )
+    except JsonableError as error:
+        error_message = "Error sending message to stream {stream}:\n{error}".format(
```

Comment:
the message should be more explicit about what's happening - it should say this is about a missed messages email reply (in its current form it sounds like it's about a regular zulip message)

Reply:
I was thinking of changing the message but it escaped my mind. Will update it.